packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.js

```diff
@@ -148,14 +153,24 @@ export default class QuanticBreadcrumbManager extends LightningElement {
   }
 
   formatDateRangeBreadcrumbValue(breadcrumb) {
+    const data = getFromStore(this.engineId, 'timeFrameFacets');
     return {
       ...breadcrumb,
+      label: data[breadcrumb.field]?.label,
       values: breadcrumb.values.map(range => ({
         ...range,
         value: this.formatDateRange(range.value),
```

Comment:
Regarding the date range, we also need to support the custom labels specified on timeframe components.

I couldn't implement the support in the breadcrumb when implementing the Timeframe facet because the formatting function (in the Timeframe facet) couldn't be shared with the BreadcrumbManager component. The compromise I made was to share the default formatting between the two components.

Now that we have a store, we should share the formatting function through the store so any custom logic is reflected on both sides.